cognitedata/reveal
# feat: Show information about currently entered 360 image

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  


#### Jira ticket :blue_book:
https://cognitedata.atlassian.net/browse/REV-842


## Description :pencil:


**Changes:**
- Added `getImageMetadata` to `Image360.`


## How has this been tested? :mag:
Using the site_id `XOM-HCU1_north_v02` for dated images, `Hibernia_RS2` and `helideck-site-2` for undated images.
On mobile (Chrome and Samsung browser(?)) and PC (Chrome & Firefox).

## Test instructions :information_source:
Add `console.log(selectedEntity.getImageMetadata());` to `onImageEntered` in `Image360UI`. Then:

1. Load one of the site_ids above.
2. Enter any image.
3. Check the console and see the information about the image that was just entered.


## Checklist :ballot_box_with_check:
